packages/web/src/pages/upload-page/components/EditPage.js

```diff
@@ -69,58 +69,65 @@ class EditPage extends Component {
       return Object.values(newInvalidTracksFields[i]).every((f) => !f)
     })
 
-    this.setState({
-      invalidTracksFields: newInvalidTracksFields
-    })
+    this.setState(
+      {
+        invalidTracksFields: newInvalidTracksFields
+      },
+      () => {
```

Comment:
smallest nit, would we want to create a helper function that we pass here? just avoids another nest